test/jstests/__init__.py

```diff
@@ -0,0 +1,47 @@
+from . import (
+    basic,
+    calc,
+    empty_return,
+    morespace,
+    strange_chars,
+    operators,
+    unary,
+    array_access,
+    parens,
+    assignments,
+    comments,
+    precedence,
+    call,
+    getfield,
+    branch,
+    switch,
+    for_loop,
+    for_empty,
+    for_in,
+    do_loop,
+    while_loop,
+    label,
+    func_expr,
+    object_literal,
+    try_statement,
+    with_statement,
+    debug,
+    unshift
+)
```

Comment:
Well, it's kind of represents the progress. At least for now. Most of the tests at the end are skipped yet.